README.md

```diff
@@ -451,54 +451,83 @@ This section describes how to secure the NATS server, including authentication,
 
 The NATS server supports single and multi-user/client authentication. See also the [server authentication](http://nats.io/documentation/server/gnatsd-authentication/) documentation.
 
-**Single user authentication**
+**Single-user Authentication**
 
-For single-user authentication, you can start the NATS server with authentication enabled by passing in the required credentials on the command line, or by passing in a token.
+For single-user authentication, you can start the NATS server with authentication enabled by passing in the required credentials on the command line.
 
 ```
-gnatsd --user foo --pass bar
+gnatsd --user derek --pass T0pS3cr3t
 ```
+You can also enable single-user authentication and set the credentials in the server configuration file as follows:
 
 ```
-gnatsd -auth 'S3Cr3T0k3n!'
+authorization {
+  user:        derek
+  password: T0pS3cr3t
+  timeout:  1
+}
 ```
 
 Clients can connect using:
 
 ```
-nats://foo:bar@localhost:4222
+nats://derek:T0pS3cr3t@localhost:4222
 ```
 
+**Token-based Authentication**
+
+A token is a unique identifier of an application requesting to connect to NATS. You can start the NATS server with authentication enabled by passing in the required token on the command line.
+
 ```
-nats://S3Cr3T0k3n!@localhost:4222
+gnatsd -auth 'S3Cr3T0k3n!'
 ```
 
-You can also enable single-user authentication and set the credentials in the server configuration file as follows:
+You can also enable token-based authentication and set the credentials in the server configuration file as follows:
 
 ```
 authorization {
-  user:     derek
-  password: T0pS3cr3t
-  timeout:  1
+  # You can generate the token using /util/mkpasswd.go
```

Comment:
Instead, I'd suggest something instead like
`# cleartext is supported but it is recommended you encrypt tokens with util/mkpasswd.go`